src/docs/gensfen.md

```diff
@@ -0,0 +1,57 @@
+# Gensfen
+
+`gensfen` command allows generation of training data from self-play in a manner that suits training better than traditional games. It introduces random moves to diversify openings, allows reduced pruning, disabling of TT for less interference between searches, and fixed depth evaluation.
+
```

Comment:
Could you please describe that it is recommended to set `PruneAtShallowDepth` to false.